### PR TITLE
GH-109190: Copyedit 3.12 What's New: asyncio

### PR DESCRIPTION
**Pull request by AA-Turner**
- Standardise Itamar's name

* Issue: gh-109190

----
:books: Documentation preview :books:: https://cpython-previews--109661.org.readthedocs.build/en/109661/whatsnew/3.12.html#asyncio

### CODE REVIEW

File: Doc/whatsnew/3.12.rst [hunk 1]
```diff
@@ -566,7 +566,7 @@ asyncio
 * Added :func:`asyncio.eager_task_factory` and :func:`asyncio.create_eager_task_factory`
   functions to allow opting an event loop in to eager task execution,
   making some use-cases 2x to 5x faster.
-  (Contributed by Jacob Bower & Itamar O in :gh:`102853`, :gh:`104140`, and :gh:`104138`)
+  (Contributed by Jacob Bower & Itamar Ostricher in :gh:`102853`, :gh:`104140`, and :gh:`104138`)
```
Comment (itamaro) [inline Doc/whatsnew/3.12.rst]:
```suggestion
  (Contributed by Jacob Bower & Itamar Oren in :gh:`102853`, :gh:`104140`, and :gh:`104138`)
```

File: Doc/whatsnew/3.12.rst [hunk 2]
```diff
@@ -983,7 +983,7 @@ Optimizations
   (Contributed by Serhiy Storchaka in :gh:`91524`.)
 
 * Speed up :class:`asyncio.Task` creation by deferring expensive string formatting.
-  (Contributed by Itamar O in :gh:`103793`.)
+  (Contributed by Itamar Ostricher in :gh:`103793`.)
```
Comment (itamaro) [inline Doc/whatsnew/3.12.rst]:
```suggestion
  (Contributed by Itamar Oren in :gh:`103793`.)
```

**Review by itamaro — DISMISSED** [review]
thanks for this @AA-Turner !
I have changed my name recently, so let's go with the current name :)

**Comment by AA-Turner** [comment]
The other instances need updating, will do when back on a
computer.

A

**Comment by hugovk** [comment]
Allow me to help :)

**Review by AA-Turner — COMMENTED** [review]
Thanks Hugo! (GH doesn't let me approve the new changes as it's my own PR)

A

**Pull request merged**